### PR TITLE
Task/dat 55 move utils to own project

### PR DESCRIPTION
**Pull request by hb0**

### CODE REVIEW

File: datacapturing/src/main/java/de/cyface/datacapturing/DiskConsumption.java [hunk 1]
```diff
@@ -1,175 +0,0 @@
-package de.cyface.datacapturing;
```
Comment (hb0) [inline datacapturing/src/main/java/de/cyface/datacapturing/DiskConsumption.java]: moved to utils (sub module)

File: datacapturing/build.gradle [hunk 2]
```diff
@@ -101,7 +101,9 @@ android {
 dependencies {
     implementation "androidx.annotation:annotation:$rootProject.ext.androidxAnnotationVersion"
     implementation "androidx.appcompat:appcompat:$rootProject.ext.androidxAppCompatVersion"
-    implementation project(":utils")
+
+    // Cyface dependencies
+    implementation 'de.cyface:android-utils:1.0.1'
```
[the remark this answers is not in the document]
Comment (hb0) [reply datacapturing/build.gradle]: Should work anywhere, see "FilesChanged" Tab, `./build.gradle`:

```
        maven {
            // Repository for Cyface libraries
            url 'https://dl.bintray.com/cyface-de/android-utils'
        }
```

File: datacapturing/src/main/java/de/cyface/datacapturing/backend/PlaceholderNotificationBuilder.java [hunk 3]
```diff
@@ -48,14 +48,14 @@
...
-final class PlaceholderNotificationBuilder {
+public final class PlaceholderNotificationBuilder {
     /**
      * Creates the placeholder notification for the provided {@code Context}.
      *
      * @param context The Android {@code Context} to create the notification for.
      * @return The notification to show, while the {@link DataCapturingBackgroundService} is loading.
      */
-    static Notification build(final @NonNull Context context) {
+    public static Notification build(final @NonNull Context context) {
```
[the remark this answers is not in the document]
Comment (hb0) [reply datacapturing/src/main/java/de/cyface/datacapturing/backend/PlaceholderNotificationBuilder.java]: see my comments in my new review below where I commented all lines where something got public in this PR

[the remark this answers is not in the document]
Comment (muthenberg) [reply datacapturing/src/main/java/de/cyface/datacapturing/backend/PlaceholderNotificationBuilder.java]: Hm. That is better. Maybe you could move the classes you think could be moved. I know this is additional work, but we will probably never do it if we don't do it now.

**Comment by hb0** [comment]
See newest commit: I moved the mentioned Methods/Classes to the Utils library, thus, had to upgrade the utils library and upgraded/cleaned up other dependencies to even out all library dependencies.

**Pull request merged**